### ISSUE TITLE
No Syntax highlighting for code block of a Markdown cell type

### ISSUE THREAD

**Issue by zunayed** [HappenApps/Quiver#70]
typing something like this in a markdown cell 

```
a = tuple(range(1000))
b = list(range(1000))

a.__sizeof__() # 8024
b.__sizeof__() # 9088
```

yields this output: 
![screenshot](http://i.imgur.com/noWYjhI.png)

I assumed writing markdown codeblocks in the notebook could be parsed similar to how github markdown works.

**Comment by nschneid** [comment]
I take it this is a fenced code block. Agreed, it would be nice if Quiver supported these (if only for pasting Markdown from other sources), though they can be approximated with a code cell.

**Comment by ylian** [comment]
Tried when working on 2.1.1. It's easy to render it as a code cell, but live preview causes serious flashes. Because this fake code cell is regenerated on every keystroke.

Leave this open since a solution is still possible.

**Comment by ylian** [comment]
I found a way to implement this properly. This will be released in the next update.

**Comment by ylian** [comment]
This is implemented in Quiver 3.0, released today.
